src/index.html

```diff
@@ -0,0 +1,127 @@
+<html lang="pl">
+<head>
+  <meta charset="UTF-8">
+  <title>Github-browser</title>
+</head>
+<body>
+
+<nav class="navbar">
+  <div class="navbar-brand">
+    <div class="navbar-item">
+      <div class="field has-addons">
+        <div class="control">
+          <input class="input username" type="text" placeholder="enter github username">
+        </div>
+        <div class="control">
+          <button class="button is-info load-username">
+            Load
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div class="navbar-item">
+      <div class="loader is-hidden" id="spinner"></div>
+    </div>
+
+  </div>
+</nav>
+
+
+<div class="container">
+  <div class="columns is-1-tablet">
+    <div class="profile-container column is-one-third">
+
+      <h2 class="subtitle is-4">Profile</h2>
```

Comment:
I don't see any H1 tag, so we can't just start with H2. They must be in correct order